.github/workflows/ubuntu.yml

```diff
@@ -126,3 +126,80 @@ jobs:
       run: |
         export OMP_NUM_THREADS=1
         mpirun -n 2 Examples/Physics_applications/laser_acceleration/PICMI_inputs_3d.py
+
+  build_qed_table_tools:
```

Comment:
My thoughts: we could save CI time by
- adding the build of the tools to an existing test

and simplify the run logic by:
- creating a script that we can call in this CI part instead of the lines added in the `.yml`

Reply:
I have converted the test into a bash script. Which test to you want me to add it to ?